README.md

```diff
@@ -246,6 +240,68 @@ skip-tree = [
 ]
 ```
 
+## Crate advisories - `cargo deny check advisories`
+
+### Use Case - Detecting security vulnerabilities
+
+Security vulnerabilities are generally considered "not great" by most people, luckily rust has a great [advisory database](https://github.com/RustSec/advisory-db) which cargo-deny can use to check that you don't have any crates with (known) security vulnerabilities.
+
+### Use Case - Detecting unmaintained crates
+
+The [advisory database](https://github.com/RustSec/advisory-db) also contains advisories for unmaintained crates which in most cases users will want to avoid in favor of more active crates.
+
+### The `[advisories]` section
+
+Contains all of the configuration for `cargo deny check advisories`
+
+#### The `db-path` field
+
+Path to the local copy of advisory database's git repo (default: ~/.cargo/advisory-db)
+
+#### The `db-url` field
+
+URL to the advisory database's git repo (default: https://github.com/RustSec/advisory-db)
```

Comment:
When reading this was a bit unclear to me if these are required, optional or why/when one would want or need to set them

Reply:
Yah I should mark them as optional, I just supported it because cargo audit also supports it. And you could have private internal databases.

Reply:
Make sense!